### PR TITLE
feat: [CCM-5834]: added connector details in analytics drawer

### PR DESCRIPTION
**Pull request by rishik30**
##### Screenshots:

Screenshot 2022-04-13 at 4 00 18 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by rishik30** [comment]
retrigger coverage

**Comment by rishik30** [comment]
trigger cypress

File: src/modules/75-ce/components/COGatewayList/COGatewayAnalytics.tsx [hunk 1]
```diff
@@ -7,13 +7,13 @@
 
 import React, { useEffect, useState } from 'react'
 import { isEmpty as _isEmpty, defaultTo as _defaultTo } from 'lodash-es'
-import { Switch, Tab } from '@blueprintjs/core'
+import { IconName, Switch, Tab } from '@blueprintjs/core'
```
Comment (rajarshee-harness) [inline src/modules/75-ce/components/COGatewayList/COGatewayAnalytics.tsx:10]: Should't `IconName` be imported from `@wings-software/uicore`

Comment (rishik30) [reply src/modules/75-ce/components/COGatewayList/COGatewayAnalytics.tsx:10]: yes, missed that

**Review by rajarshee-harness — APPROVED** [review]

**Review by riyasyash — APPROVED** [review]

**Pull request merged**